### PR TITLE
Updates SUMMARY.md

### PR DESCRIPTION
**Pull request by liatsade**
Change the menu order - put chapters under "Being Gizra employee" folder.

introduction_ _gitbook_editor

### CODE REVIEW

**Comment by liatsade** [comment]
@RachelBaram Please confirm and merge.

**Comment by RachelBaram** [comment]
@liatsade If we are fixing the menu order...

1. How about changing "Training" folder to "Technical Training" - so it will be clear of how it is different from "Being a Gizra Employee"
2. Maybe change "How and what to document" to "Work Processes" and move "How to submit a PR.." and "Pull Requests" chapters under there.  
3. Under "Being a Gizra Employee" will include only soft skills/information. That means moving "Thinking of Quitting" under there and erasing "Work life balance" folder. Will this change the url? I know that Amitai shared this chapter on twitter and I don't want to break the link.

To summarize, 3 folders:
1. Technical
2. Processes
3. Soft skills

What do you think?
